scripts/rotate-aws-creds.sh

```diff
@@ -0,0 +1,214 @@
+#!/bin/bash
+
+ENVIRONMENT=$1
+CLUSTER_ID=$2
+
+# make sure we have no state for the aws cli
+unset AWS_PROFILE
+unset AWS_ACCESS_KEY_ID
+unset AWS_SECRET_ACCESS_KEY
+export AWS_DEFAULT_REGION=us-east-1
```

Comment:
should have a comment like 
// This is a mandatory value to be set.
// It only has to be any valid region, not reflecting the region the cluster is in as the IAM interaction is globaal